src/mds/MDSAuthCaps.cc

```diff
@@ -373,6 +373,89 @@ bool MDSAuthCaps::parse(CephContext *c, std::string_view str, ostream *err)
   }
 }
 
+bool MDSAuthCaps::merge(MDSAuthCaps newcap)
```

Comment:
This method does not take into account uid/gid. I understand that you mentioned about the bug where uid/gid cannot be a part of the auth caps w/ fsname. So, will uid/gid be taken into account in that fix?

Reply:
`fs authorize` doesn't accept uid and gids as parameters. So there's no need for this method to consider them.

Reply:
Sure. Please ensure its taken care in the PR where uid/gid can be part of the auth caps with file system name.